client/utils/consoleUtils.js

```diff
@@ -16,11 +25,17 @@ export const hijackConsole = `var iframeWindow = window;
   methods.forEach( function(method) {
     iframeWindow.console[method] = function() {
       originalConsole[method].apply(originalConsole, arguments);
-  
       var args = Array.from(arguments);
+      var toString = Object.prototype.toString;
```

Comment:
to do deep copying, could you use the `lodash` method?

Reply:
I think we may not be able to do this because the code here will be run directly in the browser and cannot use the third-party libraries, right?